src/common/line_entry.rs

```diff
@@ -1,17 +1,29 @@
-use crate::common::*;
+use std::rc::Rc;
+
 use comment::Comment;
 
+use crate::common::*;
+
 #[derive(Clone, Debug, PartialEq)]
 pub struct LineEntry {
     pub number: usize,
-    pub file: FileEntry,
+    pub file: Rc<FileEntry>,
     pub raw_string: String,
 
     /// Used in ExtraBlankLineFixer
     pub is_deleted: bool,
 }
 
 impl LineEntry {
+    pub fn new(number: usize, file: Rc<FileEntry>, raw_string: String, is_deleted: bool) -> Self {
```

Comment:
```suggestion
    pub fn new(number: usize, file: Rc<FileEntry>, raw_string: String) -> Self {
```
The `is_deleted` field should be `false` by default.